husky_description/urdf/dual_arm_husky.urdf.xacro

```diff
@@ -482,7 +482,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   </joint>
   <!-- Connect PTU to Husky bulkhead -->
   <joint name="husky_ptu_joint" type="fixed">
-    <origin rpy="0 0 0" xyz="0.365 0 0.221"/>
```

Comment:
Ah, I figured it out! This offset which caused the whole PTU to float needed to be added to the mount_link for the bumblebee to be placed correctly. The value I put in is correct for this one, I will add another PR with the corrected mount link once I've figured the other two offsets out